mediaflow_proxy/extractors/mixdrop.py

```
@@ -0,0 +1,27 @@
+    async with httpx.AsyncClient(proxy=settings.proxy_url if use_request_proxy else None) as client:
+        response = await client.get(d, headers=headers, follow_redirects=True, timeout=30)
+        [s1, s2] = re.search(r"\}\('(.+)',.+,'(.+)'\.split", response.text).group(1, 2)
+        schema = s1.split(";")[2][5:-1]
+        terms = s2.split("|")
+        charset = string.digits + string.ascii_letters
+        d = dict()
+        for i in range(len(terms)):
+            d[charset[i]] = terms[i] or charset[i]
+        final_url = "https:"
+        for c in schema:
+            final_url += d[c] if c in d else c
```

Ruff

Use `d.get(c, c)` instead of an `if` block

Replace with `d.get(c, c)`

(SIM401)